azure_functions_worker/testutils_lc.py

```diff
@@ -181,11 +193,15 @@ def spawn_container(self,
         run_process = subprocess.run(args=run_cmd,
                                      stdout=subprocess.PIPE,
                                      stderr=subprocess.PIPE)
+
         if run_process.returncode != 0:
             raise RuntimeError('Failed to spawn docker container for'
                                f' {image} with uuid {self._uuid}.'
                                f' stderr: {run_process.stderr}')
 
+        # Remove once worker version 4.0.0 is released
```

Comment:
Add an issue for us to remind this.

This is released though - right?

Reply:
Its in prerelease. https://github.com/Azure/azure-functions-python-worker/releases.

Added a task for tracking.